src/Lykke.Service.PayInternal.Core/Domain/Transfer/IAddressAmount.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Lykke.Service.PayInternal.Core.Domain.Transfer
+{
+    /// <summary>
+    /// Abstract definition for entity describing what amount of asset should be transfered from/to the mentioned address.
+    /// </summary>
+    public interface IAddressAmount
```

Comment:
Why AssetId is not specified in this contract?

Reply:
Currently we have only btc transfers. It is set implicitly on application service layer.

Let's assume this to be Core Comment-2.